avaframe/com1DFA/DFAfunctionsCython.pyx

```diff
@@ -442,9 +442,11 @@ cpdef (double, double) computeEntMassAndForce(double dt, double entrMassCell,
   rhoEnt: float
     entrainement density
   """
-  cdef double width, ABotSwiped, areaEntrPart
+  cdef double width, ABotSwiped
   # compute entrained mass
   cdef double dm = 0
+  cdef double areaEntrPart = 0
```

Comment:
For this (and all following) instances: It is better to use `0.0` instead of `0` for initialization of float variables. This way the compiler cannot "optimize" it to integer...